cmd/builder/cmd/root.go

```diff
@@ -70,8 +70,8 @@ func Execute() error {
 
 	// the distribution parameters, which we accept as CLI flags as well
 	cmd.Flags().BoolVar(&cfg.SkipCompilation, "skip-compilation", false, "Whether builder should only generate go code with no compile of the collector (default false)")
-	cmd.Flags().StringVar(&cfg.Distribution.ExeName, "name", "otelcol-custom", "The executable name for the OpenTelemetry Collector distribution")
-	cmd.Flags().StringVar(&cfg.Distribution.LongName, "description", "Custom OpenTelemetry Collector distribution", "A descriptive name for the OpenTelemetry Collector distribution")
+	cmd.Flags().StringVar(&cfg.Distribution.Name, "name", "otelcol-custom", "The executable name for the OpenTelemetry Collector distribution")
```

Comment:
I think it's OK for now, but I would like to avoid breaking compatibility once we have the first release from this repo. We should then do a two-stage change, deprecating first and removing later.

Reply:
Not a breaking change, since Config is "internal" struct.

Reply:
Flags are not changed, config yaml is not changed

Reply:
True, sorry, I guess I shouldn't make code reviews in the evening :-)